api/src/main/resources/messages_fr.properties

```diff
@@ -0,0 +1 @@
+spa.title=Frontend SPA
```

Comment:
Please remove all classes that are not explicitly needed.  The SDK creates a bunch of "helpful" base classes for a common use case, but these should be deleted before committing if they are not needed.